app/src/main/java/pl/droidsonroids/droidsmap/feature/room/business_logic/RoomFeatureUseCase.kt

```diff
@@ -1,5 +1,9 @@
 package pl.droidsonroids.droidsmap.feature.room.business_logic
 
-class RoomFeatureUseCase : RoomFeatureBoundary {
-    //no-op
+import pl.droidsonroids.droidsmap.feature.office.business_logic.OfficeFeatureBoundary
+
+open class RoomFeatureUseCase(private val officeFeatureBoundary: OfficeFeatureBoundary) : RoomFeatureBoundary {
+    override fun onRoomPerspectiveGained() {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
Tak wygląda kod TODO:
```kotlin
public class NotImplementedError(message: String = "An operation is not implemented.") : Error(message)

public inline fun TODO(): Nothing = throw NotImplementedError()

public inline fun TODO(reason: String): Nothing = throw NotImplementedError("An operation is not implemented: $reason")
```
Więc sama treść "not implemented" jest już zawarta niejawnie, ale z 2 strony nie widać tego w kodzie apki.